docs/central-upgrade.rst

```diff
@@ -7,7 +7,9 @@ We release new versions of Central regularly. We recommend that you upgrade as s
 
 Start by reviewing upgrade notes for all versions between your current version and the one you are upgrading to. You can skip upgrades and directly install the latest version as long as you make sure to follow all relevant upgrade instructions.
 
-* Central v1.5, v2022.3: no upgrade notes
+* :ref:`Central v2022.3 <central-upgrade-2022.3>`: update your NGINX configuration if you have disabled or customized Sentry
+* :ref:`Central v1.5.3 <central-upgrade-1.5.3>`: fix errors with ``git pull`` if you have disabled or customized Sentry
+* Central v1.5.0: no upgrade notes
```

Comment:
It seems potentially confusing to mention patch versions like v1.5.0 here but nowhere else. But I'm not sure how else to approach it. I don't think we want to list every patch version without upgrade notes.

Reply:
I think we should have a single section per version and include special instructions for patches there. Once the patches are out, it wouldn't make sense for someone to upgrade to the .0. And in fact it's not easy to do so. I'll follow up with a proposal.